src/utils/emojis.js

```diff
@@ -80,7 +79,15 @@ const EMOJI_MAP = new Map([
   ['halloween', ':ghost:'],
   ['fire', ':fire:'],
   ['hot', ':fire:'],
+  ['heat', ':fire:'],
   ['burn', ':fire:'],
+  ['prison', ':gun:'],
+  ['Lucky', ':raised_hands:'],
```

Comment:
Mind making this lowercase? The REGEX is case insensitive so functionally it doesn't matter. It's just nice to be consistent.

Reply:
Sure, I made a separate pull request.  How do you feel about adding non standard emoji's  (ie party_parrot)

Will they fall back to a default?

Reply:
Unfortunately it won't fall back to our defaults (the music notes / headphones) so I think we should stick with standard emojis